README.md

```diff
@@ -0,0 +1,38 @@
+# Simple Node.js Demo
+
+This repository contains a simple Node.js application for demonstration purposes.
+
+## Dependency
+
+- [Node.js](https://nodejs.org/)
+
+## How to Run
+
+1. Clone the repository to your local machine:
+
+    ```bash
+    git clone https://github.com/your-username/simple-node-demo.git
+    ```
+
+2. Navigate to the project directory:
+
+    ```bash
+    cd newton-school-lessons
+    ```
+
+3. Run the application using Node.js:
+
+    ```bash
+    node server.js
+    ```
+
+   This will start the server, and you should see output indicating that the server is running.
+
+4. Open your web browser and visit [http://localhost:3000](http://localhost:3000) to see the application in action.
```

Comment:
Not necessary, the server will run only at port 3000. Can you make this line more generic?

Reply:
Hi! The port was specified at the all server.js to run at port 3000. Did I misunderstand that part, sir? Sorry, I'm a beginner at this.

Reply:
I meant to say that, not every node.js server runs on 3000, the port can be any number. Instead you can put something like $PORT in the doc and add a note that in this repo it is 3000.